sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -993,6 +993,30 @@ object functions {
     Lead(e.expr, Literal(offset), Literal(defaultValue))
   }
 
+  /**
+   * Window function: returns the value that is the `offset`th row of the window frame
+   * (counting from 1), and `null` if the size of window frame is less than `offset` rows.
+   *
+   * This is equivalent to the nth_value function in SQL.
+   *
+   * @group window_funcs
+   * @since 3.0.0
```

Comment:
Here should be changed too. You could wait for some feedback from @hvanhovell before making more changes.

Reply:
@HyukjinKwon Thanks for your reminder. I'm wating @hvanhovell here.